lib/bibliothecary/parsers/go.rb

```diff
@@ -140,6 +144,10 @@ def self.parse_go_sum(file_contents)
         deps.uniq
       end
 
+      def self.parse_go_resolved(file_contents)
+        JSON.parse(file_contents).map { |dep| { name: dep["Path"], requirement: dep["Version"], type: 'runtime' } }
```

Comment:
I think we want to skip the ones that are `Main`

Reply:
I'll add a filter in here